index.html

```diff
@@ -0,0 +1,161 @@
+          <a class="top-link" href="#">program</a>
+          <a class="top-link" href="#">Join</a>
+          <a class="top-link" href="#">Sponsor</a>
+          <a class="top-link" href="#">News</a>
+        </div>
+        <div class="hamburger" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <div class="main-bg absolute"></div>
+    <div class="main-text">
+      <h1>Welcome, English learners.</h1>
+      <h class="green-text">Join the PCC to conquer public speaking forever.</h>
+    </div>
+    <div class="scnd-text">
+      <p>
+        Join over three hundred students to learn and practice presentation skills and public
+        speaking. Compete aganist your colleage and win prestigious prizes.
+      </p>
+    </div>
+    <h2>2024.02.04 ~ 28</h2>
+    <span class="EventLocation">@ ThecEnglish Club, Alexandria, Egypt</span>
+  </section>
+
+  <section class="program-container">
+    <h1>Main Program</h1>
+    <div class="mid">
+      <hr class="line"/>
+    </div>
+  <div class="cards-div">
+  <div class="card">
+    <img src="./images/about.png" alt="Icon of About IU">        
+    <h4>About PCC</h4>
+    <p>
+        PCC stands for "Presentation Course and Competition"
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/studyprograms.png" alt="Icon of Study programmes at IU">        
+    <h4>Program</h4>
+    <p>
+        Choose your trainer. Compete aganist other teams.
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/studyingAtIU.png" alt="Icon ">        
+    <h4>Career</h4>
+    <p>
+        Learn competent career bulding skills.
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/scolarship.png" alt="Icon of Scholarship at IU">        
+    <h4>Certificate</h4>
+    <p>
+        Earn certificate of completion. 
+    </p>
+  </div>
+  <div class="card">
+    <img src="./images/qustions-answers.png" alt="Icon of Q&A session">        
+    <h4>Q&A session</h4>
+    <p>
+        Do you need help with your decision? Your personal student advisor is there!
+    </div>
+</div>
+  <button class="register" type="button">Register for the event</button>
+  <p class="see-program">See Whole Program</p>
+</section>
+
+<section class="speakers">
+  <h3>Featured Speakers</h3>
+  <div class="mid">
+    <hr class="line"/>
+  </div>
+  <div class="list"></div>
+  <div class="more-btn">
+      <span>MORE</span> 
+      <i class="fas fa-chevron-down"></i>
+  </div>
+</section>
+
+<section class="partners-conatainer-desk">
+  <h3>Partners</h3>
+  <div class="mid">
+    <hr class="line"/>
+  </div>
+  <div class="Partners-logos">
+      <i class="fab fa-aws fa-4x"></i>
+      <i class="fab fa-google fa-4x"></i>
+      <i class="fab fa-microsoft fa-4x"></i>
+      <i class="fab fa-reddit fa-4x"></i>
+      <i class="fab fa-facebook fa-4x"></i>
+    </div>
```

Comment:
- [ ] Excellent work on including your partners in the section! However, I kindly request that you **align the colors of the icons in your partners section with the [provided design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**. Please make the necessary adjustments to **ensure consistency and harmony with the overall visual aesthetic**.👏

   ### Your design:
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/da730377-30ad-41e1-866e-b97d49e0cbac)

   ### Given design:
   ![image](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/4598f317-7c6c-4f64-9d6e-342db3cff641)